### ISSUE TITLE
Accessibility: Incorrect text alternatives

### ISSUE THREAD

**Issue by jason-me** [stakwork/sphinx-tribes-frontend#1048]
## Issue
The lightning bolt, keys, and sun decorative icons announce the incorrect text alternative.

## Expected Behavior
- [WCAG 2.2 AA 1.1.1 Non-text Content](https://www.w3.org/TR/WCAG22/#non-text-content) requires that decorative images are hidden from screen reader users and that meaningful images have appropriate alt-text.
- If the image is deemed to be meaningful ensure an appropriate description, if the image is deemed decorative (i.e. conveys no meaning) hide the descritpion from screen readers 
- Screen reader users expect to hear a meaningful description which mathces the image or to not hear the image at all if it is purely decorative.
## Acceptance Criteria
- [x] Correct the alt-text on the second set of 3 icons to announce a proper description
- [x] Test with a screen reader JAWS, VoiceOver, NVDA, Orca, Talkback, etc. 
- [x] Ensure the image is announced correctly if meaningful, or not announced at all if decorative

_Linked to tracking issue: #456_

**Comment by jason-me** [comment]
Pending closure via the following PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140

**Comment by jason-me** [comment]
Merged in PR: https://github.com/stakwork/sphinx-tribes-frontend/pull/1140
